packages/core/app/mirage/routes/dashboard.js

```diff
@@ -43,17 +45,45 @@ export default function() {
   });
 
   this.get('/dashboards', ({ dashboards }, request) => {
+    let dashboardObject;
     let idFilter = request.queryParams['filter[dashboards.id]'];
+    let queryFilter = request.queryParams['filter[dashboards]'];
 
     // Allow filtering
     if (idFilter) {
       let ids = idFilter.split(',');
-      dashboards = dashboards.find(ids);
+      dashboardObject = dashboards.find(ids);
```

Comment:
could we abstract this function out to `utils` and then pull it in and use it here and in reports?